feat: add local release drafter CLI (#1684)
release-drafter/release-drafter: README.md

          config-name: release-drafter.yml # the default, loads '.github/release-drafter.yml'
```

> ### ✍️ Local CLI

Reviewer: Would avoid emojis 

```suggestion
### Local CLI
```

and also locate that section a bit lower

starting point. It reads authentication from `GITHUB_TOKEN`, `GH_TOKEN`, or the
GitHub CLI.

```sh
npx release-drafter owner/repository
```

From a source checkout, use `npm run cli -- owner/repository` instead.

Use `--from` to override the starting commitish and `--release-version` to
override the resolved version:

```sh
npx release-drafter owner/repository --from v8.0.0 --release-version 9.0.0
```

The target commitish defaults to the repository's default branch; override it with
`--to`. The CLI uses the same config loading as the action, including the fallback
to the organization's `.github` repository. Use `--config` to select another
config or `--dry-run` to print the generated release without creating or updating
it.

`--name` and `--tag` override the resolved release name and tag, mirroring the
action's `name` and `tag` inputs:

```sh
npx release-drafter owner/repository --tag v9.0.0 --name 'Release 9.0.0'
```

By default, the CLI creates or updates a draft. Use `--publish` to publish it,
optionally with `--prerelease` or `--latest`:

```sh
npx release-drafter owner/repository --publish --prerelease
npx release-drafter owner/repository --publish --latest
```

> Each release-mode flag accepts an optional `true` or `false` value, so

Reviewer: ```suggestion
The `--publish`, `--prerelease`, and `--latest` flags each accept an optional `true` or `false` value, so
```
